### PR TITLE
DOT-4683: Remove unused helpers

### PR DESCRIPTION
**Pull request by olyans**

### CODE REVIEW

File: includes/class-wc-youcanpay-helper.php [hunk 1]
```diff
@@ -157,7 +104,11 @@ public static function get_minimum_amount() {
 	 * @param string $setting The name of the setting to get.
 	 */
 	public static function get_settings( $method = null, $setting = null ) {
-		$all_settings = null === $method ? get_option( 'woocommerce_youcanpay_settings', [] ) : get_option( 'woocommerce_youcanpay_' . $method . '_settings', [] );
+		if (null === $method) {
+			$all_settings = get_option( 'woocommerce_youcanpay_settings', [] );
+		} else {
+			$all_settings = get_option( 'woocommerce_youcanpay_' . $method . '_settings', [] );
+		}
```
Comment (AmineAlyate) [inline includes/class-wc-youcanpay-helper.php:111]: I think u can use arrow function condition

File: includes/class-wc-gateway-youcanpay.php [hunk 2]
```diff
@@ -115,59 +115,21 @@ public function init_form_fields() {
...
-			data-currency="' . esc_attr( strtolower( get_woocommerce_currency() ) ) . '"
-		>';
+		echo '<div id="youcanpay-payment-data">';
 
 		if ( $this->sandbox_mode ) {
-			/* translators: link to YouCan Pay testing page */
-			$description .= ' ' . sprintf( __( 'SANDBOX MODE ENABLED. In sandbox mode, you can use the card number 4242424242424242 with 112 CVC and 10/24 date or check the <a href="%s" target="_blank">Testing YouCan Pay documentation</a> for more card numbers.',
-					'youcan-pay' ),
-					'https://pay.youcan.shop/docs#testing-and-test-cards' );
+			$text        = __( 'SANDBOX MODE ENABLED. In sandbox mode, you can use the card number 4242424242424242 with 112 CVC and 10/24 date or check the <a href="%s" target="_blank">Testing YouCan Pay documentation</a> for more card numbers.',
```
Comment (AmineAlyate) [inline includes/class-wc-gateway-youcanpay.php:126]: remove empty space

**Comment by AmineAlyate** [comment]
I think it wold be better if we use the sprintf for the concatenation

File: includes/class-wc-youcanpay-helper.php (hunk 1 above)
Comment (olyans) [reply includes/class-wc-youcanpay-helper.php:111]: @AmineAlyate  I didn't understand please give me an example

Comment (olyans) [reply includes/class-wc-youcanpay-helper.php]: https://github.com/NextmediaMa/youcan-pay/pull/4/commits/6f5cdd0780c9d961d0516777c83ebd0d7b27fbe0

File: includes/class-wc-gateway-youcanpay.php (hunk 2 above)
Comment (olyans) [reply includes/class-wc-gateway-youcanpay.php:126]: https://github.com/NextmediaMa/youcan-pay/pull/4/commits/6f5cdd0780c9d961d0516777c83ebd0d7b27fbe0

**Review by AmineAlyate — APPROVED** [review]
Technical review LGTM :heavy_check_mark:

**Pull request merged**